docs/meshcloud.profile.md

```diff
@@ -35,3 +35,5 @@ After a successful registration the next step is to Login into your federated Me
 ## Profile
 
 Navigate to the **Profile** Page via the navigation on the top right of the Meshpanel. Here you can find all your personal information stored in the Meshportal. You can also expand an area on the bottom of the page, that shows you all systems related to the Meshportal, that also persist your personal information. In general the Meshportal tries to store as little of your personal information as possible in the different systems.
+
+Here you can also manage your subscriptions if you wish to receive emails for any [platform notifications](administration.platforms.md).
```

Comment:
d: You should split up the profile page to the two section "User information" and "Platform Subscriptions". Also mention that you can subscribe to platform subscriptions per platform instance. An explanation of why the severity level can be selected and what this exactly means (e.g. that the severity info also includes warn and error) would also be helpful.